data/mods/CRT_EXPANSION/items/crt_toolarmor.json

```
@@ -151,7 +158,13 @@
     "turns_per_charge": 1,
     "revert_to": "crt_em_vest",
     "use_action": { "type": "transform", "menu_text": "Turn off armor", "msg": "CRIT EM powering off…", "target": "crt_em_vest" },
-    "artifact_data": { "charge_type": "ARTC_SOLAR", "effects_worn": [ "AEP_ALL_UP", "AEP_INT_DOWN", "AEP_PER_DOWN", "AEP_SPEED_UP" ] },
+    "//": "Legacy artifact data is set for str +2 and speed up",
+    "relic_data": {
+      "passive_effects": [
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "add": 6 } ] },
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 10 } ] }
```

```suggestion
        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 20 } ] }
```
the legacy AEP_SPEED_UP is +20